### ISSUE TITLE
Question about the recurrence of this paper

### ISSUE THREAD

**Issue by violin0847** [Pose-Group/FAMI-Pose#5]
With the source code here, I can only reach the level of mAP 82.2% in the PoseTrack2017 validation set,but it is 84.8% in the author's paper (three rounds of experiments have been done at present. Only PoseTrack2017 training set was used during training). Can anybody reach mAP 84.8%  in PoseTrack2017 validation set with this repository?

**Comment by WEIZHIHONG720** [comment]
@violin0847  hi, Can you run this code?

**Comment by WEIZHIHONG720** [comment]
@violin0847  Hi, I can't reproduce the results either, are you still at 82.2% mAP?
